docs/source/guides/building_bentos.rst

```diff
@@ -0,0 +1,238 @@
+   python:
+      packages:
+         - scikit-learn
+         - pandas
+
+Python Packages
+===============
+
+Whether you're using pip or conda, you can specify which Python packages
+to include in your Bento by configuring them in ``bentofile.yaml``.
+
+Python Options
+--------------
+
+There are two ways to specify packages in the Bentofile. First,
+we can list packages like below. When left without a version,
+pip will just use the latest release.
+
+.. code:: yaml
+
+   python:
+     packages:
+        - numpy
+        - "matplotlib==3.5.1"
+
+The user needs to put all required python packages for the Bento Service in
+a ``requirements.txt``. For a project, you can run ``pip freeze > requirements.txt``
+to generate a requirements file to load with BentoML.
+
+.. code:: yaml
+
+   python:
+     requirements_txt: "requirements.txt"
+
+Additionally, there are more fields that can help manage larger projects.
+
+.. code:: yaml
+
+   python:
+     requirements_txt: "requirements.txt"
+     lock_packages: False
+     index_url: "https://example.org/"
+     no_index: False
+     trusted_host: "localhost"
+     find_links:
+        - "https://test.org/"
+     extra_index_url:
+        - "https://test.org/"
+     pip_args: "--quiet"
+     wheels:
+        - "./libs/my_package.whl"
+
+By default, when the BentoML service generates package requirements
+from the Bentofile, the package versions will be locked for easier
+reproducibility.
+
+If the ``requirements.txt`` includes locked packages, or a configuration
+you need, set the ``lock_packages`` field to False.
+
+If you're maintaining a private pip wheel, it can be included
+with the ``wheels`` field.
+
+If the wheel is hosted on a local network without TLS, you can indicate
+that the domain is safe to pip with the ``trusted_host`` field.
+
+Conda Options
+-------------
+
+Similarly to PyPi, you can use Conda to handle dependencies.
+
+.. code:: yaml
+
+   conda:
+     dependencies:
+        - "scikit-learn==1.2.0"
+        - numpy
+        - nltk
+     channels:
+        - "conda-forge"
+
+Here, we need the conda-forge repository to install numpy with conda.
+The ``channels`` field let's us specify that to the BentoML service.
+
+In a preexisting environment, running ``conda export`` will generate
+an ``environment.yml`` file to be included in the ``environment_yml``
+field.
+
+.. code:: yaml
+
+   conda:
+     environment_yml: "environment.yml"
+
+Docker Options
+--------------
+
+BentoML makes it easy to deploy a Bento to a Docker container.
+
+Here's a basic Docker options configuration.
+
+.. code:: yaml
+
+   docker:
+     distro: debian
+     gpu: True
+     python_version: "3.8.9"
+     setup_script: "setup.sh"
+
+For the ``distro`` options, you can choose from 5.
+
+- debian
+- amazonlinux2
+- alpine
+- ubi8
+- ubi7
+
+This config can be explored from `BentoML's Docker page <https://hub.docker.com/r/bentoml/bento-server>`_.
+
+The `gpu` field will also allocate a GPU in the Docker.
```

Comment:
when gpu is true, bentoml uses a base image that has cudnn and Nvidia driver installed